aries-site/src/pages/foundation/accessibility.mdx

```diff
@@ -1,12 +1,16 @@
 import { GuideSection, VideoSection } from '../../examples/foundation';
+import { getPageDetails, getRelatedContent, nameToPath } from '../../utils';
+import { CardGrid } from '../../components';
+
+export const wcagChecklistCard = [getPageDetails('WCAG Checklist')];
```

Comment:
Using `export` in MDX is the convention for using variables/JSX in the same .mdx file that it's created in. If we want to avoid any variables/JSX being created directly in .mdx files, then I agree with moving it to the `examples` folder.